utils/constants.tsx

```diff
@@ -857,7 +857,7 @@ export const NotificationLevels = {
     ALL: 'all',
     MENTION: 'mention',
     NONE: 'none',
-};
+} as const;
```

Comment:
do we need this?

Reply:
In order to use it as a type, we need to assert as `const`. For more info, please check [official doc](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions)